cmd/crc/cmd/stop.go

```diff
@@ -22,15 +26,46 @@ var stopCmd = &cobra.Command{
 	},
 }
 
-func runStop(arguments []string) {
+var (
+	force bool
+)
 
+func runStop(arguments []string) {
 	stopConfig := machine.StopConfig{
 		Name: constants.DefaultName,
 	}
 
+	killConfig := machine.KillConfig{
+		Name: constants.DefaultName,
+	}
+
+	if force {
+		killVM(killConfig)
+		errors.Exit(0)
+	}
+
 	commandResult, err := machine.Stop(stopConfig)
+	if err != nil {
+		// Here we are catching the error "VM Failed to gracefully shutdown, try the kill command"
+		// from the driver side and if it exist then we ask user input for power off operation.
+		if strings.Contains(err.Error(), "kill") {
+			var userInput string
+			output.OutF("Do you want to force power off [y/n]: ")
+			fmt.Scan(&userInput)
+			if strings.ToLower(userInput) == "y" {
+				killVM(killConfig)
+				errors.Exit(0)
+			}
+		}
+		errors.ExitWithMessage(1, err.Error())
+	}
+	output.Out(commandResult.Success)
+}
+
+func killVM(killConfig machine.KillConfig) {
+	commandResult, err := machine.Kill(killConfig)
```

Comment:
Isn't this a `PowerOff` ? Kill sounds more like a `kill -9` of the VM process.

Reply:
Also, if the user chooses `n`, then it defaults to the message `[VM Failed to gracefully shutdown, try the kill command]` even though in the meantime the VM did shut down gracefully.

Reply:
That is what driver is returning `VM Failed to gracefully shutdown, try the kill command`, I am still think that just have `-f` for stop no user interaction at all, we will increase the timelimit around `120sec` on driver side.

Reply:
I saw you made those changes